src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,60 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  constructor(name, health = 100) {
+    super(name, health);
+
+    this.name = name;
+    this.health = health;
```

Comment:
These assignments are redundant because the `name` and `health` properties are already set in the parent `Animal` class constructor when you call `super(name, health)`. You can remove lines 18 and 19.